### ISSUE TITLE
Files in Subsubfolders are ignored

### ISSUE THREAD

**Issue by Hirse** [globexdesigns/brackets-jscs#8]
When I put the `.jscs.json` into the Project Root, files in the `js` folder will get linted as expected:
![jscs_1](https://cloud.githubusercontent.com/assets/2564094/3102100/912eab78-e641-11e3-93c9-49827baf6265.PNG)
But files in the `js_inner` folder are ignored:
![jscs_2](https://cloud.githubusercontent.com/assets/2564094/3102258/7899da08-e644-11e3-990e-c0af0748e600.PNG)
To get linting for that file, I have to copy `.jscs.json` into `js` (or `js_inner`):
![jscs_3](https://cloud.githubusercontent.com/assets/2564094/3102284/c188ef60-e644-11e3-9419-3f310c9321c0.PNG)
However when I delete the `.jscs.json` from the project root, only files in the direct subfolders are affected:
![jscs_4](https://cloud.githubusercontent.com/assets/2564094/3157799/9958a548-eafa-11e3-871c-bc458813bd2c.PNG)
![jscs_5](https://cloud.githubusercontent.com/assets/2564094/3157816/d66277a2-eafa-11e3-844b-a54b1c058b1f.PNG)

---

In summary, I would like to have linting for all files in the project, so I don't have to copy the settings.
